src/markup/html.rs

```diff
@@ -0,0 +1,221 @@
+//! Utils for working with the [HTML message style][spec].
+//!
+//! [spec]: https://core.telegram.org/bots/api#html-style
+
+use reqwest::Url;
+
+use crate::types::{User, UserId};
+
+/// Applies the bold font style to the string.
+///
+/// Passed string will not be automatically escaped because it can contain
+/// nested markup.
+pub fn bold(s: &str) -> String {
+    format!("<b>{s}</b>")
+}
+
+/// Applies the italic font style to the string.
+///
+/// Passed string will not be automatically escaped because it can contain
+/// nested markup.
+pub fn italic(s: &str) -> String {
```

Comment:
How about adding `#[must_use]` to these functions?

Reply:
@WaffleLapkin what do you think?

Reply:
I agree, these functions (just as a lot of already existing), should be marked with `#[must_use]`

Reply:
Marking as resolved, I've added `#[must_use]` on every function.